device/samples/cloud_to_device_message.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+'use strict';
+
+require('dotenv').config();
```

Comment:
And this does?

Reply:
dotenv allows users to create an environment variables file, and then automatically import the environment variables